### PR TITLE
Allow registration of content bundles in GitHubHint

### PR DESCRIPTION
**Pull request by jacogr**

### CODE REVIEW

File: js/src/dapps/githubhint/Application/application.js [hunk 1]
```diff
@@ -147,53 +188,81 @@ export default class Application extends Component {
...
+      this.lookupHash(`https://codeload.github.com/${repo}/zip/${commit}`);
+    });
+  }
+
+  onChangeRepo = (event) => {
+    let repo = event.target.value;
+    const repoError = null;
+
+    if (!repoError) {
+      repo = repo.replace('https://github.com/', '');
```
Comment (ngotchac) [inline js/src/dapps/githubhint/Application/application.js]: `repoError` will always be `null` here

File: js/src/dapps/githubhint/Application/application.js [hunk 2]
```diff
@@ -238,8 +353,10 @@ export default class Application extends Component {
     this.setState({ fromAddress: addresses[index] });
   }
 
-  lookupHash () {
-    const { url, instance } = this.state;
+  lookupHash (url) {
+    const { instance } = this.state;
+
+    console.log(`lookupHash ${url}`);
```
Comment (ngotchac) [inline js/src/dapps/githubhint/Application/application.js]: Check that the url isn't empty. Otherwise, errors will pop-up all the time

Comment (ngotchac) [inline js/src/dapps/githubhint/Application/application.js]: Even better, that the input is an actual URL (no need to try for `htt`)

Comment (ngotchac) [inline js/src/dapps/githubhint/Application/application.js]: (Same for the Content hash)

**Comment by ngotchac** [comment]
Another thing : if I register a URL but then reject the transaction, I won't be able to use the dApp anymore

**Comment by jacogr** [comment]
Ok, will have to log an issue for that - non-existent new functionality, very basic to start off with. (I actually would also like multi-reg - at least for my use-cases actually need it)

**Pull request merged**